src/components/Toolbar/index.js

```diff
@@ -263,6 +263,10 @@ export default class Toolbar extends Component {
                 <td>Space bar</td>
                 <td>Flip orientation between down/across</td>
               </tr>
+              <tr>
+                <td><code>Home</code> OR <code>End</code></td>
```

Comment:
at some point we may want to create a separate section for advanced shortcuts like Home/End/[/]/Tab, so that new users aren't overwhelmed with the size of the list

Reply:
Good idea! While we're doing that we have some instructions that render wrong. For example,

```html
                  <code>[</code> and
                  <code>]</code> OR
                  <code>Shift</code> with arrow keys
```

ends up rendering as `[ and] ORShift with arrow keys` so we should do something like

```html
                  <code>[</code> and <code>]</code> OR <code>Shift</code> with arrow keys
```

while making that change.

Reply:
Nice find and agreed!

Would you be able to include building the advanced section as part of this PR, or would you rather wait until that is done? I'm not entirely comfortable adding more keyboard shortcuts until we can be confident that they won't confuse new users, so that does mean it is somewhat blocking for your feature.